content/courses/program-security/signer-auth.md

```diff
@@ -199,7 +199,7 @@ performed.
 #### Use Anchor’s `#[account(signer)]` constraint
 
 While in most cases, the `Signer` account type will suffice to ensure an account
-has signed a transaction, the fact that no other ownership or type checks are
+has signed a transaction, the fact that no other ownership or type of checks are
```

Comment:
No. We want to say that there are no other checks regarding **ownership** or **types**.

If you like, you can re-word this sentence to make it clearer.